### PR TITLE
Guild/systemchannel

### PR DESCRIPTION
**Pull request by devsnek**
**Please describe the changes this PR makes and why it should be merged:**

**Semantic versioning classification:**  
- [x] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/structures/Guild.js [hunk 1]
```diff
@@ -756,6 +756,12 @@ class Guild {
     return this.edit({ afkChannel }, reason);
   }
 
+  /**
+   * Edit the system channel of the guild.
+   * @param {ChannelResolvable} systemChannel The new AFK channel
```
Comment (KingDGrizzle) [inline src/structures/Guild.js]: AFK Channel?

Comment (iCrawl) [inline src/structures/Guild.js]: Yes, its not new it was just never there.

File: src/structures/Guild.js [hunk 2]
```diff
@@ -304,6 +310,24 @@ class Guild {
...
+   * @type {?VoiceChannel}
+   * @readonly
+   */
+  get afkChannel() {
+    return this.client.channels.get(this.afkChannelID);
+  }
+
+  /**
+   * System channel for this guild
+   * @type {?GuildChannel}
```
Comment (SpaceEEC) [inline src/structures/Guild.js]: That is always a TextChannel (or null), the client only allow selecting those and manually patching it to a voice one removes it.

Comment (devsnek) [inline src/structures/Guild.js]: just trust me on this one :3

Comment (SpaceEEC) [inline src/structures/Guild.js]: If you say so

**Pull request merged**